Training_Preparation.md

```diff
@@ -0,0 +1,403 @@
+
+The result is, as expected, the shell descriptor that our query previously specified.
+
+
+#### Task 2: Creating a New Shell Descriptor
+
+In this task, we will learn, how to create a new shell descriptor. 
+
+
+#### a) Request a JWT token for DTR Authentication / Authorization
+
+See above
+
+#### b) Create a Shell Descriptor
+
+
+With a POST request to the Shell-Descriptors API, you can create any number of new Shell Descriptors. The request body is structurally mostly identical to the output of the Shell Descriptor request from the previous task.
+
+
+#### c) Create a Shell Descriptor, Part II
+
+Please copy the response body from Task 1 d) and manually insert it here as the request body. 
+
+Please slightly modify the UUID in the "id" field at the top (change one digit).
+
+Then execute the request and review the response.
+
+Also, please note, that this is a business-wise key check. So if you want to try this multiple times,
+you will need to reset the id to a new value.
+
+
+### Task 3: Interaction between EDC and DTR
+
+In this task we will learn, how the DTR of external partners can be utilised via the means of Tractus-X-connector based dataspace. 
+
+As a customer, we're now going to look up a material twin and then get the actual submodel data using the EDC for negotiation. 
+The flow can be seen in the following diagram. 
+
+![Api Flow](docs/architecture/img/06-api-flow-detailed.svg)
+
+This flow is also described in the Digital Twin KIT and is used for standards 
+relying on the Digital Twin KIT. Whenever you're wondering about the EDC steps, look them up in the 
+[management api walkthrough of the EDC](https://github.com/eclipse-tractusx/tractusx-edc/tree/main/docs/usage/management-api-walkthrough).
+
+#### a) Catalog Query for the DTR Asset
+
+Please look at the request body. We use a filter there that should yield a unique result. The filter should find exactly 
+the asset that the other partner has created for his DTR on his own DTR. 
+
+
+#### b) Negotiation for a Contract over the Asset
+
+The request body contains parts of the catalog entry received in the previous request. It has been copied in the 
+previous step (automatically via Postman Scripting). Please revisit a) and the script. If you like to, try to build 
+it on your own. Following the [Connector Kit](https://eclipse-tractusx.github.io/docs-kits/category/connector-kit), 
+this policy node needs to be **exactly the same as offered by the supplier**.
+
+
+#### c) Querying the Negotiation Status
+As a response, we receive the contract ID.
+
+
+#### d) Requesting a Pull Transfer
+We provide the contract ID.
+
+
+#### e) Requesting an EDR
+
+The EDR contains the data (authentication information and URL) with which we can subsequently access the asset. 
+The authentication token also only has a validity duration of a few minutes. Please note that the EDR may need to be 
+renewed by re-executing this request. 
```

Comment:
This is the first time people will work with EDR. Please add more details here and describe in more detail what an EDR is, why it is used, etc.